### PR TITLE
I have added instructions for how to download the nightly artifact from Azure

### PR DESCRIPTION
**Pull request by RMJay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,16 @@ Windows nightlies built on Azure
 | **Android SDK** | [![Build Status](https://dev.azure.com/compnerd/windows-swift/_apis/build/status/android%20SDK?branchName=master)](https://dev.azure.com/compnerd/windows-swift/_build/latest?definitionId=4?branchName=master) |
 
 ### Getting Started
+#### Downloading the nightlies
+
+1. We assume you are reading these instructions on the Azure DevOps page for windows-swift at <https://dev.azure.com/compnerd/windows-swift>.
+2. Choose `Pipelines` > `builds` from the left of the dashboard.
+3. Click the most recent successfull build.
+4. At the top right of this page to the right of where it says `All logs` there is a three dot button. Under this choose `Artefacts` > `toolchain` to download the nightly toolchain to your machine.
```
Comment (compnerd) [inline README.md]: Typo: `Artefacts` -> `Artifacts`.

File: README.md [hunk 2]
```diff
@@ -10,10 +10,16 @@ Windows nightlies built on Azure
 | **Android SDK** | [![Build Status](https://dev.azure.com/compnerd/windows-swift/_apis/build/status/android%20SDK?branchName=master)](https://dev.azure.com/compnerd/windows-swift/_build/latest?definitionId=4?branchName=master) |
 
 ### Getting Started
+#### Downloading the nightlies
+
+1. We assume you are reading these instructions on the Azure DevOps page for windows-swift at <https://dev.azure.com/compnerd/windows-swift>.
+2. Choose `Pipelines` > `builds` from the left of the dashboard.
+3. Click the most recent successfull build.
```
Comment (compnerd) [inline README.md]: Typo `successfull` -> `successful`.

File: README.md [hunk 3]
```diff
@@ -10,10 +10,16 @@ Windows nightlies built on Azure
...
+4. At the top right of this page to the right of where it says `All logs` there is a three dot button. Under this choose `Artefacts` > `toolchain` to download the nightly toolchain to your machine.
 
 #### Installing the nightlies
 
-1. Extract the toolchain artifact.  We assume that the toolchain will be installed to `C:\Library`.  Extracting it to that location should give you a hierarchy that looks like `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swiftc.exe`.
+1. Extract the toolchain artefact.  We assume that the toolchain will be installed to `C:\Library`.  Extracting it to that location should give you a hierarchy that looks like `C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\bin\swiftc.exe`.
```
Comment (compnerd) [inline README.md]: Typo: `artefacts` -> `artifacts`.

**Comment by compnerd** [comment]
Can you please rebase (and squash) your patch?

**Comment by RMJay** [comment]
I believe I have have incorporated your feedback and squashed the commits as you requested. Thank you for your patience whilst I have been learning features of git unfamiliar to me.

**Pull request merged**